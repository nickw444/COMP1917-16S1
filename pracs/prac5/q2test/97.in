97
-4 -8 2 3 8 2 -3 -1 10 -3 10 -1 8 -9 -3 9 -5 -8 2 -1 -7 8 2 -6 -7 -7 2 -6 2 -7 -2 0 10 1 1 7 -9 6 -1 -6 -9 -6 6 3 -8 6 -6 10 2 -7 -6 -9 10 2 -10 0 -10 -3 0 0 -5 6 -7 6 -9 1 4 -8 -3 6 -7 -2 9 7 1 -5 -9 -2 0 -8 -6 0 8 5 3 6 -8 -2 9 -3 -3 -7 -5 -3 4 9 0
