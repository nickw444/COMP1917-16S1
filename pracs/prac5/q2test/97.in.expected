average of array: -0.659794
